src/components/container/DocumentLibraryPreview/DocumentLibraryPreview.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import './DocumentLibraryPreview.css';
+import { createAllLibraryDocumentsLoader, createLibraryDocumentSorter, formatNumberForLocale, language, LibraryDocument, LibraryDocumentSurveySpecification, useInitializeView } from '../../../helpers';
+import { Action, Button } from '../../presentational';
+import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
+import MyDataHelps from '@careevolution/mydatahelps-js';
+import { faCamera } from '@fortawesome/free-solid-svg-icons';
+import { createPreviewDocuments, DocumentLibraryPreviewPreviewState } from './DocumentLibraryPreview.previewData';
+
+export interface DocumentLibraryPreviewProps {
+    previewState?: 'loading' | DocumentLibraryPreviewPreviewState;
+    surveySpecification: LibraryDocumentSurveySpecification;
+    documentLibraryViewBaseUrl: string;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+/**
+ * This component acts as an informational CTA to allow participants to open the DocumentLibraryView.
+ * When no documents have been uploaded, the CTA shortcuts directly to the document upload survey.
+ */
+export default function DocumentLibraryPreview(props: DocumentLibraryPreviewProps) {
+    const [documents, setDocuments] = useState<LibraryDocument[]>();
+
+    const applyPreviewState = (previewState: 'loading' | DocumentLibraryPreviewPreviewState): void => {
+        setDocuments(previewState !== 'loading' ? createPreviewDocuments(previewState) : undefined);
+    };
+
+    useInitializeView(() => {
+        if (props.previewState) {
+            applyPreviewState(props.previewState);
+            return;
+        }
+
+        createAllLibraryDocumentsLoader().load(props.surveySpecification, false).then(documents => {
+            setDocuments(documents.sort(createLibraryDocumentSorter('date', 'desc')));
+        });
+    }, [], [props.previewState]);
+
+    if (!documents) return null;
```

Comment:
Why return nothing instead of a placeholder card with a spinner?  If the card is going to take up space once it loads (regardless of wither there are existing documents or not) there's less visual bouncing if the base card is at least present on the page.